samples/subsys/video/capture_to_lcd/boards/mini_stm32h743.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CLOCK_STM32_MCO1_SRC_HSI48=y
+CONFIG_CLOCK_STM32_MCO1_DIV=4
```

Comment:
Is MCO requested for the sample to work ?

Reply:
Yes. This board doesn't have a crystal oscillator for the camera module. The XCLK is provided by pin PA8.

![image](https://github.com/zephyrproject-rtos/zephyr/assets/27859307/e96115d8-4d0d-413d-840e-d32111411b8f)